### PR TITLE
feat: add commands to nodejs builder

### PR DESCRIPTION
**Pull request by laurentsimon**

### CODE REVIEW

File: internal/builders/nodejs/ci.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+// ciCmd runs the 'ci' command.
+func ciCmd(check func(error)) *cobra.Command {
+	var ciArguments []string
+	var ciDirectory string
+
+	c := &cobra.Command{
+		Use:   "ci",
+		Short: "Run ci command",
+		Long:  `Run ci command to install dependencies of a node project.`,
```
Comment (asraa) [inline internal/builders/nodejs/ci.go:31]: nit: add an example usage with the arguments

File: internal/builders/nodejs/main.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+func rootCmd() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "slsa-github-generator",
+		Short: "Generate SLSA provenance for Github Actions",
+		Long: `Generate SLSA provenance for Github Actions.
+For more information on SLSA, visit https://slsa.dev`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return errors.New("expected command")
```
Comment (asraa) [inline internal/builders/nodejs/main.go:43]: maybe print Usage string provided by cobra?

**Review by asraa — COMMENTED** [review]
I'm guessing this is just template work right now? If so LGTM, maybe just add some example usage or start the README.md with the expected commands.

**Comment by laurentsimon** [comment]
> I'm guessing this is just template work right now? If so LGTM, maybe just add some example usage or start the README.md with the expected commands.

correct, just the template for the command we need. I'll add README info.

File: internal/builders/nodejs/main.go (hunk 2 above)
Comment (laurentsimon) [reply internal/builders/nodejs/main.go:43]: it does it automatically, it seems

**Review by ianlewis — APPROVED** [review]

**Pull request merged**